source/_integrations/device_tracker.markdown

```diff
@@ -114,3 +115,34 @@ The `device_tracker.see` service can be used to manually update the state of a d
 | `gps`                  |      yes | If you're providing a location, for example `[51.513845, -0.100539]` |
 | `gps_accuracy`         |      yes | The accuracy of the GPS fix |
 | `battery`              |      yes | The battery level of the device |
+
+### Device Tracker with a Toggle Button
+You want to "track" a guest e.g. for a guest room, who cannot be tracked manual.
+A solution can be to create a Helper Toggle Button. In the example the button will be called "Guest"
+Then you create the following automation:
+```yaml
+alias: Guest as Button
+description: ""
+trigger:
+  - platform: state
+    entity_id:
+      - input_boolean.guest
+condition: []
+action:
+  - choose:
+      - conditions:
+          - condition: template
+            value_template: "{{ states('input_boolean.guest') == 'on' }}"
+        sequence:
+          - service: device_tracker.see
+            data:
+              location_name: home
+              dev_id: guest
+    default:
+      - service: device_tracker.see
+        data:
+          dev_id: guest
+          location_name: not_home
+mode: single
+```
+After you Toggle the Button the first time a device_tracker.guest will apear which you can use e.g. for person (with the name Guest)
```

Comment:
I'm going to friendly decline this example for our documentation.

While I understand it could be helpful for some cases or setups, it is a custom workaround with helpers in the end, which adds a lot of complexity for a very small use case. For example, one could be using `input_boolean.guest` directly instead.

The community forums, however, is a great place to share these types of examples 👍 

../Frenck